src/test/java/uk/gov/hmcts/reform/adoption/service/CoreCaseDataServiceTest.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.reform.adoption.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.adoption.request.RequestData;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.reform.ccd.client.model.Event;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+import uk.gov.hmcts.reform.idam.client.IdamClient;
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.adoption.service.CoreCaseDataService.CASE_TYPE;
+import static uk.gov.hmcts.reform.adoption.service.CoreCaseDataService.JURISDICTION;
+import static uk.gov.hmcts.reform.adoption.service.CoreCaseDataService.START_CASE_EVENT;
+
+@ExtendWith(MockitoExtension.class)
+class CoreCaseDataServiceTest extends BaseTest {
```

Comment:
a bit wired to extends to have access just to static fields, unless it is fo future to share behaviour

Reply:
maybe it is better to extract this if there is ever a need rather than assuming? But yes I assume ObjectMapper and anything else that is fairly common to be defined in basetest